.gitignore

```diff
@@ -1 +1,20 @@
-flagged
\ No newline at end of file
+# PyCharm Projects
+.idea/
+
+# Vitural Env
+venv*/
```

Comment:
Does each python project have to have an environment?

Reply:
Yes, a very common way to work is to have a clean and personal env. This way, reproduction is very good and projects dont interfere. 

Please clone the repo to a new location on your machine, try to install pyenv (its REALLY good -- you will be able to remove pythons from your machine after this), follow the process of installing a specific python version using pyenv, then setting a specific version, then preparing it and then creating a venv, the activating and finally, installing all the packages. This will be a good QA to my README.md (which i DIDNT test).

Reply:
Will be done!